crates/strategies/blend-strategies/src/liquidation_strategy.rs

```diff
@@ -334,10 +309,11 @@ impl BlendLiquidator {
                             .unwrap(),
                         )
                         .unwrap();
+
                     info!(" New pending bad debt fill: {:?}", pending_fill.clone());
-                    self.pending_fill.push(pending_fill);
+                    self.pending_fill.push(pending_fill.clone());
                     //we only care about lot here
-                } else {
+                } else if auction_type == 2 {
```

Comment:
Does this code path not handle user liquidations?

Reply:
New auction events are only for bad debt and interest auctions. User liquidations come in their own event